doc/Comments.xml

```diff
@@ -157,12 +158,14 @@ can be used to set the label of the chapter to <C>Chapter_</C><A>label</A>.
 
 Additionally, the chapter will be stored as <C>_Chapter_</C><A>label</A><C>.xml</C>.
 
-
+The <C>@ChapterTitle</C> <A>title</A> command
+can be used to set a heading for the chapter that is different com <A>name</A>
```

Comment:
Typo: "com" -> "from" (I assume?)

Also, I am confused now: If I do this:
```
@Chapter Foo
@ChapterLabel Bar
@ChapterTitle Qux
```
Then I know `Bar` is used for the filename and the label. And `Qux` is used for the chapter heading. But what is `Foo` now used for?

Reply:
For opening the chapter again in the next file to put more sections in it.